2.Views/WebAppPages/Settings/ProfilePage.cs

```diff
@@ -39,4 +39,13 @@ public class ProfilePage
     [Element("New Password", ElementType.TextField)]
     [Locator(LocatorType.Id, "TextPwNew")]
     public TextField NewPassTextField => UIElementFactory.GetElement(elementName: "New Password", "Profile Page");
+
+    [Element("Time Zone", ElementType.Button)]
+    [Locator(LocatorType.Id, "DropDownTimezone")]
+    public Button TimeZoneOption => UIElementFactory.GetElement("Time Zone", "Profile Page");
```

Comment:
Do we need the UIElementFactory on this getter? I think `{ get; }` is enough.
Same for "Hawaiian Time" element.